apps/frontend-pwa/src/pages/editProfile.tsx

```diff
@@ -145,85 +164,179 @@ function EditProfile() {
+                              root: 'bg-red-600 border-red-700 text-white',
+                            }}
+                            onClick={async () => {
+                              await deleteParticipantAccount()
+                              window?.location.reload()
+                            }}
+                          >
+                            {t('shared.generic.confirm')}
+                          </Button>
+                        }
+                        onSecondaryAction={
+                          <Button onClick={() => setDeleteModalOpen(false)}>
+                            {t('shared.generic.cancel')}
+                          </Button>
+                        }
+                        trigger={
+                          <Button
+                            onClick={(): void => setDeleteModalOpen(true)}
+                          >
+                            {t('shared.generic.delete')}
+                          </Button>
+                        }
                       >
-                        {t('shared.generic.save')}
-                      </Button>
+                        {t('pwa.profile.deleteProfileConfirmation')}
+                      </Modal>
                     </div>
                   </div>
+                </div>
 
-                  <div className="flex-1 order-1 space-y-2 md:order-2">
-                    {Object.keys(AVATAR_OPTIONS).map((key) => (
-                      <div className="flex flex-row items-center" key={key}>
-                        <div className="flex-1">
-                          <p className="font-bold">{t(`pwa.avatar.${key}`)}</p>
-                        </div>
+                <div className="md:bg-slate-50 md:p-4 rounded space-y-2">
+                  <H3 className={{ root: 'border-b' }}>Avatar</H3>
+                  <div className="flex flex-col md:flex-row gap-4 md:gap-8">
+                    <div className="flex-1">
+                      <BigHead
+                        // @ts-ignore
+                        className="border-b-4 md:h-48 border-primary-80 w-full"
+                        eyebrows="raised"
+                        faceMask={false}
+                        lashes={false}
+                        mask={false}
+                        clothing={values.clothing}
+                        hat="none"
+                        graphic="none"
+                        accessory={values.accessory}
+                        body="chest"
+                        skinTone={values.skinTone}
+                        clothingColor={values.clothingColor}
+                        eyes={values.eyes}
+                        hair={values.hair}
+                        hairColor={values.hairColor}
+                        mouth={values.mouth}
+                        facialHair={values.facialHair}
+                      />
+                    </div>
+                    <div className="grid grid-cols-2 md:grid-cols-3 gap-2 text-xs">
```

Comment:
Text appears to be very small on larger laptop screens / desktop screens. Maybe we could scale up the font a bit on larger screens?

Reply:
```suggestion
                    <div className="grid grid-cols-2 md:grid-cols-3 gap-2 text-xs md:text-sm">
```